README.md

```diff
@@ -255,3 +245,56 @@ You may run multiple controls:
 bundle
 rake test:integration[azure_resource_group,azurerm_virtual_machine]
 ```
+
+### Optional Components
+
+By default, rake tasks will only use core components. Optional components have
+associated integrations that will be skipped unless you enable these. We have
+the following optional pieces that may be managed with Terraform.
```

Comment:
This doesn't read clearly as the next part I go into is examples and not the optional components.